JavaCryptographicArchitecture/src/SecureRandom.crysl

```diff
@@ -27,7 +26,6 @@ EVENTS
 
 	gS: genSeed = generateSeed(_);	
 
-	ne: next(numB);
 	nB: nextBytes(next);
 	nI: randInt = nextInt();
 	Nexts := ne | nB | nI;
```

Comment:
@rakshitkr thanks for noticing this. Would you also please remove the `ne` part in this line since we are removing the `next()` method? And, could you please do the same changes to the SecureRandom rule in the BouncyCastle-JCA ruleset [here](https://github.com/CROSSINGTUD/Crypto-API-Rules/blob/master/BouncyCastle-JCA/src/SecureRandom.crysl)? Thanks very much!